### ISSUE TITLE
Uncaught TypeError: Cannot read property 'length' of null

### ISSUE THREAD

**Issue by Bennyz** [atom/tree-view#378]
1. Right click -> Add Root Folder
2. Press ESC

**Atom Version**: 0.182.0
**System**: linux 3.18.6-1-ARCH
**Thrown From**: [tree-view](https://github.com/atom/tree-view) package, v0.159.0

### Stack Trace

Uncaught TypeError: Cannot read property 'length' of null

```
At /usr/share/atom/resources/app/node_modules/tree-view/lib/tree-view.js:1006

TypeError: Cannot read property 'length' of null
  at /usr/share/atom/resources/app/node_modules/tree-view/lib/tree-view.js:1006:42
  at EventEmitter.<anonymous> (/usr/share/atom/resources/app/src/atom.js:406:16)
  at EventEmitter.emit (events.js:116:17)

```

### Commands

```
     -0:45.3 command-palette:toggle (atom-pane.pane.active)
  3x -0:40.3 core:cancel (atom-pane.pane.active)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "solarized-light-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by maxbrunsfeld** [comment]
@Bennyz are you building Atom from source? This bug is fixed in the latest tree-view version 0.160.0 which should have shipped with Atom 0.182.

**Comment by Bennyz** [comment]
@maxbrunsfeld I installed it using my package manager which builds it from source. I think the error reporter generated the report with the wrong version number. Because I just now upgraded to 0.182:

==> Software upgrade (new version) :
aur/atom-editor-git 0.181.0.r31.gd1e6dba-1 -> 0.182-1

And the bug is indeed fixed. Thanks
